### DISCUSSION TITLE
cco:Road

### DISCUSSION THREAD

**Discussion by neilotte** [CommonCoreOntology/CommonCoreOntologies#361] [Announcements]
cco:Road is presently defined: "A Transportation Infrastructure Artifact that is designed to enable Ground Vehicles to travel from one location to another."

This definition seems too broad, as it seems to also describe bridges, tunnels, traffic lights, gas stations... 

What about the following update? 
_A Transportation Infrastructure Artifact that consists of a solid platform of prepared material--including earth, rock, brick, or asphalt--designed to enable Ground Vehicles to travel on it from one location to another._

The only remaining issue is how roads are to be distinguished from other roads (e.g. Do roads continue through intersections or not? Are roads distinguished only by the names given them relative to some system of naming? Can roads branch or include forks, or would forking entail the termination of a road?) Any guidance here would be appreciated.

**Comment by nklsbckmnn** [comment]
1. There is a distinction between road and roadway (lane) in German law: 

A road is supposed to be a land area designed for vehicle and pedestrian traffic including all structures that serve this purpose.

A roadway (lane) is the part of a road that is designed for vehicle traffic.

Your current definition seems to me to better fit the latter.

2. I don’t see any reason to exclude intersections (or forks) based on the empirical usage of the term. 

In the geospatial domain, there seems to commonly be made a distinction between a fiat road/street and a road segment (or elementary street unit). It is the latter that seems to be taken to exclude intersections, to require continuous surface material and so on.

http://www.lgam.info/road-segment
https://s3.eu-west-1.amazonaws.com/static.geoplace.co.uk/downloads/NSG-Data-Entry-Conventions-and-Best-Practice-v4.1-2019.pdf

At first sight, individuating roads (fiat wholes consisting of segments) by some system of naming to me does not seem to be the worst idea.

**Comment by oliviahobai** [comment]
Moving to discussion to continue development on what it means to be a road.
